cardano-testnet/src/Testnet/Components/DReps.hs

```diff
@@ -247,15 +247,20 @@ failToSubmitTx
   => H.ExecConfig -- ^ Specifies the CLI execution configuration.
   -> AnyCardanoEra -- ^ Specifies the current Cardano era.
   -> File SignedTx In -- ^ Signed transaction to be submitted, obtained using 'signTx'.
+  -> String -- ^ Substring of the error to check for to ensure submission failed for
+            -- the right reason.
   -> m ()
-failToSubmitTx execConfig cEra signedTx = GHC.withFrozenCallStack $ do
-  (exitCode, _, _) <- H.execFlexAny' execConfig "cardano-cli" "CARDANO_CLI"
+failToSubmitTx execConfig cEra signedTx reasonForFailure = GHC.withFrozenCallStack $ do
```

Comment:
You're missing `HasCallStack` constraint on this function most likely.

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/pull/5816/commits/23c5a68a8bc324d9021350b257d4e87899697fc8